### ISSUE TITLE
Diff editor verbosity setting name should be camelcased

### ISSUE THREAD

**Issue by joyceerhl** [microsoft/vscode#183970]
https://github.com/microsoft/vscode/blob/eef30e7165e19b33daa1e15e92fa34ff4a5df0d3/src/vs/workbench/contrib/accessibility/browser/accessibilityContribution.ts#L12

Should this rather be
```ts
DiffEditor = 'accessibility.verbosity.diffEditor',
```

<img width="969" alt="image" src="https://github.com/microsoft/vscode/assets/30305945/cf756624-c711-4e45-9b77-456cda527d11">
